networks/p2p/discover/ntp.go

```diff
@@ -43,18 +44,38 @@ func (s durationSlice) Len() int           { return len(s) }
 func (s durationSlice) Less(i, j int) bool { return s[i] < s[j] }
 func (s durationSlice) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
 
+// checkClockDrift queries an NTP server for clock drifts and warns the user if
+// one large enough is detected.
+func CheckClockDriftForBoot(url, port string) {
+	drift := driftThreshold * 2
+	for drift < -driftThreshold || drift > driftThreshold {
+		time.Sleep(time.Second)
```

Comment:
Do you have to wait before the first try? Otherwise the Sleep can be moved to the end of this loop.

Reply:
resolved it. Thanks!